src/Stepper/_variables.scss

```diff
@@ -0,0 +1,19 @@
+$stepper-header-bg:                                       transparent !default;
+$stepper-header-min-height:                               5.13rem !default;
+$stepper-header-padding-y:                                .75rem !default;
+$stepper-header-padding-x:                                $spacer !default;
+$stepper-header-line-bg:                                  $light !default;
+$stepper-header-line-height:                              1px !default;
+$stepper-header-step-color:                               $primary !default;
+$stepper-header-step-bg:                                  $stepper-header-bg !default;
+$stepper-header-step-border:                              none !default;
+$stepper-header-step-min-width:                           0 !default;
+$stepper-header-step-padding:                             .25rem !default;
+$stepper-header-step-list-padding-y:                      .25rem !default;
+$stepper-header-step-list-padding-x:                      0 !default;
+$stepper-header-step-list-margin:                         0 !default;
+$stepper-header-active-step-bg:                           $gray-500 !default;
+$stepper-header-step-error-bubble-color:                  $danger !default;
+$stepper-header-step-error-bubble-box-shadow-color:       $stepper-header-step-error-bubble-color !default;
+$stepper-header-step-error-bubble-box-shadow-width:       3px !default;
```

Comment:
I would reduce the name to 5 words max. In this case readability of variable is complicated, although the idea as I understand is to make it more clear what this value means.

Reply:
It`s interesting to think. Today we don't have special rules for naming SCSS variables. I agree with your idea about the unreadability of long names in SCSS variables. Unfortunately, in this case, I can't come up with a good combination for this variable (using 5 words). Would you happen to have any suggestions for this?